tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -374,6 +376,31 @@ install_shimv2() {
 	DESTDIR="${destdir}" PREFIX="${prefix}" "${shimv2_builder}"
 }
 
+install_ovmf() {
+	ovmf_type="${1:-x86_64}"
+	tarball_name="${2:-edk2.tar.xz}"
+
+	local component_name="ovmf"
+	local component_version="$(get_from_kata_deps "externals.ovmf.${ovmf_type}.version")"
+	[ "${ovmf_type}" == "tdx" ] && component_name="tdvf"
+	install_cached_tarball_component \
+		"${component_name}" \
+		"${jenkins_url}/job/kata-containers-main-ovmf-${ovmf_type}-$(uname -m)/${cached_artifacts_path}" \
```

Comment:
Another possibility for the the job's name is `component_name` rather than `ovmf-${ovmf_type}`? What would be more flexible for future changes?

Reply:
Never mind. It would make the code more complex because we would need to convert ovmf_type -> component_name in other scripts like the one for caching.

Reply:
This one I'm not that much in favour, @wainersm.

What we'll end up having is:
* ovmf-tdx
* ovmf-sev-snp
* ovmf-x86_64 (the default one, which can also be used for `sev-es`)

I think as it's right now is very much flexible for future changes, without any other work apart from copying what's done for the tdx function.